mesonbuild/interpreter.py

```diff
@@ -1214,6 +1214,11 @@ def get_cross_property_method(self, args, kwargs):
                 return args[1]
             raise InterpreterException('Unknown cross property: %s.' % propname)
 
+
+permitted_kwargs = {'project': set(['version', 'meson_version', 'default_options', 'license', 'subproject_dir']),
+                 }
```

Comment:
__[flake8]__
* [E124] closing bracket does not match visual indentation

<!-- Comment by [SideCI](https://sideci.com) -->